pycuda/gpuarray.py

```diff
@@ -1767,6 +1767,104 @@ def make_func_for_chunk_size(chunk_size):
 
 # {{{ shape manipulation
 
+def concatenate(arrays, axis=0, allocator=None):
+
+    # impllementation is borrowed from pyopencl.array.concatenate()
+    # {{{ find properties of result array
```

Comment:
Move the [fold marker](https://vim.fandom.com/wiki/Folding#Folding_with_markers) after the docstring.

Reply:
^